exercises/numbers/power/power.js

```diff
@@ -11,6 +11,13 @@
  * @returns {number} The base raised to the power of the exopnent
  */
 function power(base, exponent) {
+  if (exponent === 0){
+    return 1
+  }
+  while ( exponent > 1) {
+    return base * power(base, exponent-1);
+  }
+  return base;
   /*
```

Comment:
You can remove these comments, they add noise to your commits and are just there to orient students.